### PR TITLE
Document the 'system' event types for 'podman events'

### PR DESCRIPTION
**Pull request by mheon**
Fixes: #4002

### CODE REVIEW

File: docs/podman-events.1.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ podman\-events - Monitor Podman events
 Monitor and print events that occur in Podman. Each event will include a timestamp,
 a type, a status, name (if applicable), and image (if applicable).  The default logging
 mechanism is *journald*. This can be changed in libpod.conf by changing the `events_logger`
-value to `file`.  Only `file` and `journald` are the accepted.
+value to `file`.  Only `file` and `journald` are the accepted. A `none` logger is also
```
Comment (TomSweeneyRedHat) [inline docs/podman-events.1.md]: s/the accepted/the accepted values/   or perhaps just accepted.

Comment (mheon) [reply docs/podman-events.1.md]: Good catch, fixed

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**